lib/dotcom/alerts.ex

```diff
@@ -8,9 +8,22 @@ defmodule Dotcom.Alerts do
 
   @stops_repo_module Application.compile_env!(:dotcom, :repo_modules)[:stops]
 
+  @type diversion_effect_t() ::
+          :detour | :service_change | :shuttle | :station_closure | :stop_closure | :suspension
+
   @typedoc "Service alerts which typically impact rider experience."
   @type service_effect_t() :: :delay | :service_change | :shuttle | :suspension | :station_closure
 
+  # A keyword list of effects and the severity level necessary to make an alert a 'diversion.'
+  @diversion_effects [
+    detour: 1,
+    service_change: 3,
+    shuttle: 1,
+    station_closure: 1,
+    stop_closure: 1,
+    suspension: 1
+  ]
```

Comment:
Not sure it's more helpful but we could probably abstract this to `  @diversion_effects @service_impact_effects ++ [detour: 1, stop_closure: 1]`
🫠 It almost adds a tiny bit of product clarity, because as far as I remember this was primarily for bus problems (detours and stop closures are more of a bus thing, hence the spec for describing subway/rail problems wouldn't need to consider them...

Reply:
Why I think it would be more confusing to try to derive them.